### ISSUE TITLE
[Bug] Unable to load Mecab.

### ISSUE THREAD

**Issue by Lenos500** [coqui-ai/TTS#3393]
### Describe the bug

When I try to launch the demo gui in colab for xtts_v2 it gives this error 
RuntimeError: ---------------------------------------------------------- Failed initializing MeCab. Please see the README for possible solutions: https://github.com/SamuraiT/mecab-python3#common-issues If you are still having trouble, please file an issue here, and include the ERROR DETAILS below: https://github.com/SamuraiT/mecab-python3/issues issueを英語で書く必要はありません。 ------------------- ERROR DETAILS ------------------------ arguments: [ifs] no such file or directory: /usr/local/lib/python3.10/dist-packages/unidic/dicdir/mecabrc ----------------------------------------------------------
Traceback:
File "/usr/local/lib/python3.10/dist-packages/streamlit/runtime/scriptrunner/script_runner.py", line 541, in _run_script
    exec(code, module.__dict__)
File "/content/Dem/Dem/Dem/Dem/applogo.py", line 4, in <module>
    from TTS.api import TTS
File "/usr/local/lib/python3.10/dist-packages/TTS/api.py", line 12, in <module>
    from TTS.utils.synthesizer import Synthesizer
File "/usr/local/lib/python3.10/dist-packages/TTS/utils/synthesizer.py", line 11, in <module>
    from TTS.tts.configs.vits_config import VitsConfig
File "/usr/local/lib/python3.10/dist-packages/TTS/tts/configs/vits_config.py", line 5, in <module>
    from TTS.tts.models.vits import VitsArgs, VitsAudioConfig
File "/usr/local/lib/python3.10/dist-packages/TTS/tts/models/vits.py", line 33, in <module>
    from TTS.tts.utils.text.characters import BaseCharacters, BaseVocabulary, _characters, _pad, _phonemes, _punctuations
File "/usr/local/lib/python3.10/dist-packages/TTS/tts/utils/text/__init__.py", line 1, in <module>
    from TTS.tts.utils.text.tokenizer import TTSTokenizer
File "/usr/local/lib/python3.10/dist-packages/TTS/tts/utils/text/tokenizer.py", line 5, in <module>
    from TTS.tts.utils.text.phonemizers import DEF_LANG_TO_PHONEMIZER, get_phonemizer_by_name
File "/usr/local/lib/python3.10/dist-packages/TTS/tts/utils/text/phonemizers/__init__.py", line 10, in <module>
    from TTS.tts.utils.text.phonemizers.ja_jp_phonemizer import JA_JP_Phonemizer
File "/usr/local/lib/python3.10/dist-packages/TTS/tts/utils/text/phonemizers/ja_jp_phonemizer.py", line 3, in <module>
    from TTS.tts.utils.text.japanese.phonemizer import japanese_text_to_phonemes
File "/usr/local/lib/python3.10/dist-packages/TTS/tts/utils/text/japanese/phonemizer.py", line 355, in <module>
    _TAGGER = MeCab.Tagger()
File "/usr/local/lib/python3.10/dist-packages/MeCab/__init__.py", line 135, in __init__
    raise RuntimeError(error_info(rawargs)) from ee

### To Reproduce

Try launching the hugging face demo in colab. 

### Environment

```shell
Google colab
Latest version of tts
All requirements installed as indicated in the hugging face demo.
```

**Comment by jbartolozzi** [comment]
I'm seeing a similar issue while trying to use `language="ja"`

**Comment by BobReal0822** [comment]
Have you solved this problem? @Lenos500 @jbartolozzi

**Comment by jbartolozzi** [comment]
I was able to get around this for my purposes by pip installing the following libs
`pip install mecab-python3 cutlet unidic-lite`

**Comment by Lenos500** [comment]
> Have you solved this problem? @Lenos500 @jbartolozzi

Yeah I solved it 2 months ago by doing pip install which is listed in the comments here.
